devlab/provision/generate_config.sh

```diff
@@ -38,7 +38,8 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    sed -i "s|<${key}>|${value}|g" ${result_config}
+    [ "$OSTYPE" != "linux-gnu" ] && sed -i .bak "s|<${key}>|${value}|g" ${result_config} && rm -f ${result_config}.bak
+    [ "$OSTYPE" == "linux-gnu" ] && sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
There's a separate patch for this already: #481, please remove these changes from this pull request.